### DISCUSSION TITLE
Does DPA-2 Finetuning Require Data Generated with the Same DFT Software and Parameters as Pre-Training?

### DISCUSSION THREAD

**Discussion by zx25j** [deepmodeling/deepmd-kit#4456] [Q&A]
Hello, everyone,
I have a question regarding DPA2 finetuning. Should the data provided for finetuning be consistent with the data used in pre-training? Specifically, is it necessary to use the same software and parameters for DFT calculations when generating the labels?

If consistency is not required, how can we ensure that the energy and forces computed by different software remain on the same scale?

Thank you for your insights!

**Comment by QuantumMisaka** [comment]
it is not required, the finetuning process can be seen (in a simpliy way) as a model transfer learning from a upstream model. 

You only to make sure your training data have consistent label

**Reply by zx25j** [reply]
Thank you so much for your helpful response! I have another question. If I use the same software and parameters as in the pre-training model for generating labels, will this reduce the amount of data required for fine-tuning and make it easier to train the model to achieve the desired performance?

Looking forward to your insights!
